ksp_plugin/fork_handle.hpp

```diff
@@ -0,0 +1,61 @@
+#pragma once
+
+#include "base/not_null.hpp"
+#include "ksp_plugin/frames.hpp"
+#include "ksp_plugin/plugin.hpp"
+#include "physics/discrete_trajectory.hpp"
+
+namespace principia {
+namespace ksp_plugin {
+
+// Owns a fork.
+// TODO(egg): templatize, move to physics, make movable, make NewFork return
```

Comment:
And let's not change NewFork, returning an object that's not moveable/copyable would be a big PITB.